LocalSettings.php

```diff
@@ -3146,8 +3147,12 @@
 			),
 		),
 		'+wikicanadawiki' => array(
-			'Founder' => array(
+			'founder' => array(
 				'read' => true,
+				'checkuser' => true,
```

Comment:
@1-2-1-2-3-Miraheze No, this is not related to the DB. We simply don't give out these permissions for privacy reasons